content/css-properties/colors.md

```diff
@@ -0,0 +1,145 @@
+
+A hexadecimal color is specified with: #RRGGBB, where the RR (red), GG (green) and BB (blue) hexadecimal integers specify the components of the color. All values must be between 00 and FF.
+
+Example:
+
+```css
+ .bg-color {
+ 	background-color: #ff0000;
+ }   /* red */
+
+```
+### RGB Colors
+
+RGB color values are supported in all major browsers.
+
+An RGB color value is specified with: rgb(red, green, blue). Each parameter (red, green, and blue) defines the intensity of the color and can be an integer between 0 and 255 or a percentage value (from 0% to 100%).
+
+Example:
+
+```css
+ .bg-color {
+ 	background-color: rgb(0, 0, 255);
+ }   /* blue */
+
+```
+
+
+### RGBA Colors
+
+RGBA color values are supported in IE9+, Firefox 3+, Chrome, Safari, and in Opera 10+.
+
+RGBA color values are an extension of RGB color values with an alpha channel - which specifies the opacity of the object.
+
+Example:
+
+```css
+ .bg-color {
+ 	background-color: rgb(255, 0, 0, 0.3);
+ }   /* red with opacity of 0.3 */
+
+```
+Note: The alpha parameter is a number between 0.0 (fully transparent) and 1.0 (fully opaque).
+
+
+### HSL Colors
+
+HSL color values are supported in IE9+, Firefox, Chrome, Safari, and in Opera 10+.
+
+HSL stands for hue, saturation, and lightness - and represents a cylindrical-coordinate representation of colors.
+
+An HSL color value is specified with: hsl(hue, saturation, lightness).
+
+Hue is a degree on the color wheel (from 0 to 360) - 0 (or 360) is red, 120 is green, 240 is blue. Saturation is a percentage value; 0% means a shade of gray and 100% is the full color. Lightness is also a percentage; 0% is black, 100% is white.
+
+Example:
+
+```css
+ .bg-color {
+ 	background-color: hsl(120, 100%, 50%);
+ }   /* green */
+
+```
+
+### HSLA Colors
+
+HSLA color values are supported in IE9+, Firefox 3+, Chrome, Safari, and in Opera 10+.
+
+HSLA color values are an extension of HSL color values with an alpha channel - which specifies the opacity of the object.
+
+An HSLA color value is specified with: hsla(hue, saturation, lightness, alpha), where the alpha parameter defines the opacity.
+
+```css
+ .bg-color {
+ 	background-color: hsl(120, 100%, 50%, 0.3);
+ }   /* green with opacity*/
+
+```
+Note: The alpha parameter is a number between 0.0 (fully transparent) and 1.0 (fully opaque).
+
+### Color Names
+
+All modern browsers support the following 140 color names. These are the basic color keyword:
+
+- Black	
+- Silver	
+- Gray	
+- White	
+- Maroon	
+- Red	
+- Purple	
+- Fuchsia	
+- Green	
+- Lime	
+- Olive	
+- Yellow	
+- Navy	
+- Blue	
+- Teal	
+- Aqua
+
+Example:
+
+```css
+ .bg-color {
+ 	background-color: aqua;
+ } 
+
+```
+
+### Considerations about the color use
+
+#### Hexadecimal Colors 
+
+Is very attractive because it’s short and simple to remember and type out. But HEX might not work for you in all situations, which is when you may wish to consider one of the other two methods. Both of which have their pluses and minuses.
```

Comment:
Can you add when RGB is not the best solution? For example, if i want a background color with .5 opacity, why should i use RGBA instead of RGB and set the opacity to 0.5?